Dockerfile

```diff
@@ -1,13 +1,13 @@
 FROM alpine:3.11
 
-RUN apk add --no-cache coreutils curl jq gfortran libc-dev cmake make python3 git
+RUN apt-get update && apt-get install -y coreutils curl jq gfortran libc-dev cmake make python3 git
```

Comment:
And particular reason why you changed this? This dockerfile is based on alpine and alpine uses apk and doesn't have `apt-get` installed. You can verify this by doing `docker build .` in the repo directory.

Reply:
No,  thank you for noticing. I got confused because gitpod uses sudo apt-get and the Dockerfile uses apk. Changed that part back.